packages/workflow/src/WorkflowErrors.ts

```diff
@@ -15,6 +15,7 @@ export class WorkflowOperationError extends ExecutionBaseError {
 
 	constructor(message: string, node?: INode) {
 		super(message, { cause: undefined });
+		this.severity = 'warning';
```

Comment:
To make sure, we want to stop reporting all `WorkflowOperationError` and `SubworkflowOperationError`? As we would lose e.g. errors like [this](https://github.com/n8n-io/n8n/blob/1055bd3762b90b013a300bd87e3fa902e902cb9e/packages/cli/src/eventbus/MessageEventBus/recoverEvents.ts#L88-L90).

Reply:
I don't think we want to report that Sentry. We should use Sentry for for tracking unhandled and actionable errors. 
If we want to keep a track of possible OOM crashes, then we should use logging or telemetry for that.

Reply:
I looked into all instance of `WorkflowOperationError` and `SubworkflowOperationError`, and they all looked like warnings to me. But if you think some of them should be reported, then we should update the constructors to accept `severity` in the `options` argument, just like how `NodeOperationError` does.